pkgs/by-name/de/devcontainer-cli/package.nix

```diff
@@ -0,0 +1,97 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, fetchYarnDeps
+, makeWrapper
+, nodejs
+, prefetch-yarn-deps
+, yarn
+, git
+, docker
+, nodePackages
+, testers
+, devcontainer-cli
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "devcontainer-cli";
+  version = "0.53.0";
+
+  src = fetchFromGitHub {
+    owner = "devcontainers";
+    repo = "cli";
+    rev = "v${version}";
+    hash = "sha256-WslXoyE+l26kiKVbwAKAYG5KkFmhT7zUY5mLTq+yP/8=";
+  };
+
+  offlineCache = fetchYarnDeps {
+    yarnLock = ./yarn.lock;
+    hash = "sha256-Ybqjx7FN2J0IzO/rdKh7X57Nw08dzGZtjKr3PXLZEcg=";
+  };
+
+  nativeBuildInputs  = [
+    makeWrapper
+    nodejs
+    prefetch-yarn-deps
+    yarn
+  ];
+
+  runtimeDependencies = [
```

Comment:
Is that even being used?

Reply:
Yes, the whole tool relies on these two for fetching, building and running its containers. It does so in this manner:

- [example of `git` usage](https://github.com/devcontainers/cli/blob/34f35c2e424dd681842c0ed22d4c4f7769673496/build/hygiene.js#L227)
- [example of `docker` usage](https://github.com/devcontainers/cli/blob/34f35c2e424dd681842c0ed22d4c4f7769673496/src/spec-node/featuresCLI/testCommandImpl.ts#L89)

Reply:
I think stdenv.mkDerivation is not using this by default.